src/Core/Domain/Customer/Group/Command/AddCustomerGroupCommand.php

```diff
@@ -75,4 +112,40 @@ public function showPrice(): bool
     {
         return $this->showPrice;
     }
+
+    /**
+     * @return ShopId[]
+     */
+    public function getShopIds(): array
+    {
+        return $this->shopIds;
+    }
+
+    private function assertIsCorrect()
+    {
+        $this->assertReductionIsEqualOrGreaterThanZero();
+    }
+
+    private function assertReductionIsEqualOrGreaterThanZero()
+    {
+        $this->assertIsPrice($this->reduction);
+    }
+
+    private function assertIsPrice(DecimalNumber $price)
+    {
+        $regexCheck = preg_match('/^[0-9]{1,10}(\.[0-9]{1,9})?$/', (string) $price);
+        if (0 === $regexCheck || false === $regexCheck) {
+            throw new InvalidReductionException();
+        }
+
+        if ($price->isNegative()) {
+            throw new InvalidReductionException();
```

Comment:
exception is missing error message. (Like reduction cannot be negative or so?)

Reply:
Used already existing exception and added another code instead